src/managers/GuildMemberManager.js

```diff
@@ -263,7 +263,7 @@ class GuildMemberManager extends BaseManager {
     user: user_ids,
     query,
     time = 120e3,
-    nonce = Date.now().toString(16),
+    nonce = Math.random().toString(16),
```

Comment:
Sadly, this is [bound to collisions](https://stackoverflow.com/questions/578700/how-trustworthy-is-javascripts-random-implementation-in-various-browsers), which is why we have crypto.

However, we have [`Snowflake.generate()`](https://github.com/discordjs/discord.js/blob/d2341654fe6ef87c6048bb5bf74c394062f3f618/src/util/Snowflake.js#L29-L47), which can generate up to 4095 different IDs in the same millisecond - very unlikely for it to hit.

Reply:
While that is true, how high are the chances of actually having a collision within the short time frame that gulld member requests nonces are valid?

Reply:
It is actually very low, but I wouldn't like to take chances :)